src/UnitySampleProject/Assets/Scripts/Prefabs/Gelato/GelatoModule.cs

```diff
@@ -0,0 +1,209 @@
+using System.Numerics;
+using System.Threading;
+using System.Threading.Tasks;
+using ChainSafe.GamingSDK.EVM.MetaMaskBrowserWallet;
+using ChainSafe.GamingWeb3.Build;
+using ChainSafe.GamingWeb3.Unity;
+using ChainSafe.GamingSdk.Gelato;
+using ChainSafe.GamingSdk.Gelato.Dto;
+using ChainSafe.GamingWeb3;
+using UnityEngine;
+using Web3Unity.Scripts.Library.Ethers.JsonRpc;
+using Contract = Web3Unity.Scripts.Library.Ethers.Contracts.Contract;
+
+public class GelatoModule : MonoBehaviour
```

Comment:
This script can be renamed to GelatoExamples just like the prefab. Also all of the example scripts are now stored at Samples folder (except for the Minter, don't know why), so it's probably a better place for this one too.